### PR TITLE
Add event meshmates lightning talk

### LINKED ISSUE #2791: [Event] Lightning Talk: MeshMates : A New Way of Expanding Open Source Community Mentorship
Please add the following event to layer5.io/events.

**Event:** Lightning Talk: MeshMates : A New Way of Expanding Open Source Community Mentorship
- Event Title: Open Source Summit North America
- Event Link: https://ossna2022.sched.com/
- Talk Description: See the talk link
- Talk Link: https://ossna2022.sched.com/event/11Pyl/lightning-talk-meshmates-a-new-way-of-expanding-open-source-community-mentorship-shivay-lamba-layer5
- Speaker: [Shivay Lamba](https://ossna2022.sched.com/speaker/shivaylamba)

---
<h4>Contributor Resources</h4>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by shivaylamba**
Signed-off-by: shivay <shivaylamba@gmail.com>

**Description**

This PR fixes #2791 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5dc9d6056997c26915cefff2a096751127a95a37 at: https://628d65f4f9906a1d84444dcb--layer5ng.netlify.app

**Comment by shivaylamba** [comment]
@kayceeDev actually the reason I added 24th was because my talk is on 24th. Are we supposed to add the event start date?

**Comment by kayceeDev** [comment]
> @kayceeDev actually the reason I added 24th was because my talk is on 24th. Are we supposed to add the event start date?

Thanks for the clarity. Looks good to me

**Review by kayceeDev — APPROVED** [review]

**Review by kayceeDev — APPROVED** [review]

**Review by leecalcote — CHANGES_REQUESTED** [review]
When we have more than one talk at the conference, it's ideal that we use the conference start date. In this case, it's probably not worth changing. What is worth changing is the image used. Ideally, it doesn't have dates or hashtags overlaid, but is just the conference logo (with colored background).

File: src/collections/events/ossna-2022/index.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+thumbnail: ./oss-na-2022.jpeg
+eurl: https://events.linuxfoundation.org/open-source-summit-north-america/
+type: Event
+published: true
+upcoming: true
+feature: true
+speakers: ['Shivay Lamba']
+---
+
+<h3>Lightning Talk: MeshMates : A New Way of Expanding Open Source Community Mentorship</h3>
```
Comment (leecalcote) [inline src/collections/events/ossna-2022/index.mdx:13]:
```suggestion
<h3>MeshMates: A New Way of Expanding Open Source Community Mentorship</h3>
```

File: src/collections/events/ossna-2022/index.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<p>
+A lot of young developers especially college students who want to contribute to open source projects feel intimidated by the lack of experience and the size of the code. For a beginner, open source projects codebase can be intimidating at first. Traditionally they may contact community members or mentors to help them understand but it is not always the most effective solution.
+
+Therefore Layer5 ( part of CNCF ) introduced a program called Meshmates. MeshMates who are members part of the Layer5 community, provide one to one mentorship to new joiners in the community, from helping them identifying areas of projects to engage within, working groups to join, and in helping community members grow in their open source and cloud native knowledge.
```
Comment (leecalcote) [inline src/collections/events/ossna-2022/index.mdx:17]:
```suggestion
Therefore Layer5 ( a CNCF member company) introduced a program called <Link to="/community/meshmates">Meshmates</Link>. MeshMates who are members part of the Layer5 community, provide one to one mentorship to new joiners in the community, from helping them identifying areas of projects to engage within, working groups to join, and in helping community members grow in their open source and cloud native knowledge.
```

Comment (leecalcote) [reply src/collections/events/ossna-2022/index.mdx:17]: This will require the import of `Link`.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by shivaylamba** [comment]
I have resolved the comments @leecalcote, but I can't find any better image as compared to the current one for the conference. 
This is another image, but not sure if it is the best one
OSS-NA-22-HeroBanner-v3-B-purple
Here;s another one: https://blogs.vmware.com/opensource/files/2022/05/open_source_summit-576x324.png

Please let me know which is the best one which could choose.

**Comment by l5io** [comment]
🚀 Preview for commit 0ae058f9cb09bd209ac004243d60e4ac4783a471 at: https://628f4019b2a6183e8d12bb94--layer5ng.netlify.app

**Comment by leecalcote** [comment]
@shivaylamba, I've made one for you. Here you go -
oss-na-2022

**Comment by shivaylamba** [comment]
Thank you very much @leecalcote I have updated the image

**Review by leecalcote — APPROVED** [review]
💥 Looking good!

**Pull request merged**